# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: security/sandbox/chromium/sandbox/win/src/target_interceptions.h
# Commit: 370c40e49230
# Full Hash: 370c40e4923073a87845b57c1f4851bacd9663da
# Author: Bob Owen <bobowencode@gmail.com>
# Date: 2019-06-12 21:42:35
# Regressor Bug: 1552160
# File Overlap Count: 2
# Description:
#   Bug 1552160 Part 1: Roll-up of chromium sandbox update and mozilla patches to get a running browser. r=jld,aklotz,tjr,bobowen
#   
#   This updates security/sandbox/chromium/ files to chromium commit 84108231f6e6e0772fb9a4643679ce76aa771e67.
#   
#   Existing and new patches applied from security/sandbox/chromium-shim/patches/with_update/ to give a compiling and mostly working browser.
# ==============================================================================

diff -r bd5ecd72f76c -r 370c40e49230 security/sandbox/chromium/sandbox/win/src/target_interceptions.h
--- a/security/sandbox/chromium/sandbox/win/src/target_interceptions.h	Wed Jun 12 21:55:59 2019 +1200
+++ b/security/sandbox/chromium/sandbox/win/src/target_interceptions.h	Wed Jun 12 11:10:48 2019 +0100
@@ -2,12 +2,12 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+#ifndef SANDBOX_WIN_SRC_TARGET_INTERCEPTIONS_H_
+#define SANDBOX_WIN_SRC_TARGET_INTERCEPTIONS_H_
+
 #include "sandbox/win/src/nt_internals.h"
 #include "sandbox/win/src/sandbox_types.h"
 
-#ifndef SANDBOX_SRC_TARGET_INTERCEPTIONS_H__
-#define SANDBOX_SRC_TARGET_INTERCEPTIONS_H__
-
 namespace sandbox {
 
 extern "C" {
@@ -15,21 +15,29 @@
 // Interception of NtMapViewOfSection on the child process.
 // It should never be called directly. This function provides the means to
 // detect dlls being loaded, so we can patch them if needed.
-SANDBOX_INTERCEPT NTSTATUS WINAPI TargetNtMapViewOfSection(
-    NtMapViewOfSectionFunction orig_MapViewOfSection, HANDLE section,
-    HANDLE process, PVOID *base, ULONG_PTR zero_bits, SIZE_T commit_size,
-    PLARGE_INTEGER offset, PSIZE_T view_size, SECTION_INHERIT inherit,
-    ULONG allocation_type, ULONG protect);
+SANDBOX_INTERCEPT NTSTATUS WINAPI
+TargetNtMapViewOfSection(NtMapViewOfSectionFunction orig_MapViewOfSection,
+                         HANDLE section,
+                         HANDLE process,
+                         PVOID* base,
+                         ULONG_PTR zero_bits,
+                         SIZE_T commit_size,
+                         PLARGE_INTEGER offset,
+                         PSIZE_T view_size,
+                         SECTION_INHERIT inherit,
+                         ULONG allocation_type,
+                         ULONG protect);
 
 // Interception of NtUnmapViewOfSection on the child process.
 // It should never be called directly. This function provides the means to
 // detect dlls being unloaded, so we can clean up our interceptions.
-SANDBOX_INTERCEPT NTSTATUS WINAPI TargetNtUnmapViewOfSection(
-    NtUnmapViewOfSectionFunction orig_UnmapViewOfSection, HANDLE process,
-    PVOID base);
+SANDBOX_INTERCEPT NTSTATUS WINAPI
+TargetNtUnmapViewOfSection(NtUnmapViewOfSectionFunction orig_UnmapViewOfSection,
+                           HANDLE process,
+                           PVOID base);
 
 }  // extern "C"
 
 }  // namespace sandbox
 
-#endif  // SANDBOX_SRC_TARGET_INTERCEPTIONS_H__
+#endif  // SANDBOX_WIN_SRC_TARGET_INTERCEPTIONS_H_